keps/sig-api-machinery/0006-apply.md

```diff
@@ -135,14 +135,17 @@ A brief list of the changes:
 The linked documents should be read for a more complete picture.
 
 ### Implementation Details/Notes/Constraints [optional]
-
+```
 (TODO: update this section with current design)
 
 What are the caveats to the implementation?
 What are some important details that didn't come across above.
 Go in to as much detail as necessary here.
 This might be a good place to talk about core concepts and how they releate.
+```
 
+* Server-side apply no longer falls back on client-side apply when the server does not have the logic for Server-side apply 
```

Comment:
I would have created a new section maybe called `kubectl` where I would have "documented" the current behavior:
```
# Kubectl

The feature is not enabled by default on kubectl, but can be used by running `kubectl apply 
--experimental-server-side ...`. If the feature is not available or enabled on the server, the 
command will fail rather than fall-back on client-side apply, since the semantics are too different.

As the feature graduates to Beta, the flag will be renamed to `--server-side`. In a later version, 
this parameter will be enabled by default. TODO: Describe a plan for switching that default.
```

Reply:
Something along these lines

Reply:
Sounds good. Ill update that tonight

Reply:
Thanks!